packages/dropdown/src/Dropdown.tsx

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable prettier/prettier */
+import React, { useState } from 'react';
+
+import { DropdownProps, SelectOptions } from './Dropdown.types';
+
+import styles from './styles/default.module.css';
+
+export const Dropdown = ({ value, onChange, options, disabled, dataTestId }: DropdownProps) => {
+  const [stateValue, setStateValue] = useState<SelectOptions | undefined>(
+    value,
+  );
+
+  const [isOpen, setIsOpen] = useState(false);
+
+  const selectOption = (option: SelectOptions) => {
+    if (onChange) {
```

Comment:
Hey, @hcafaq could you check all jest coverage tests, please?

Reply:
resolved